apps/events/models.py

```diff
@@ -344,14 +344,20 @@ def repetitions(self) -> str:
         return self.params["repetitions"]
 
     def as_string(self, as_timezone: str | None = None):
-        schedule = f"{self.name}: Every {self.frequency} {self.time_period}, {self.repetitions} times"
-        if self.time_period not in ["hour", "day"]:
-            weekday = self.next_trigger_date.strftime("%A")
-            schedule = (
-                f"{self.name}: Every {self.frequency} {self.time_period} on {weekday} for {self.repetitions} times"
-            )
+        if self.repetitions == 0:
+            schedule = f"{self.external_id}: One-off reminder"
+        else:
+            schedule = f"{self.external_id}: Every {self.frequency} {self.time_period}, {self.repetitions} times"
+            if self.time_period not in ["hour", "day"]:
+                weekday = self.next_trigger_date.strftime("%A")
+                schedule = (
+                    f"{self.external_id}: Every {self.frequency} {self.time_period} on {weekday} for "
+                    f"{self.repetitions} times"
+                )
+
         next_trigger = pretty_date(self.next_trigger_date, as_timezone=as_timezone)
-        return f"{schedule} (next trigger is {next_trigger})"
+        from_system = self.action is not None
+        return f"{schedule} with next trigger at {next_trigger}. (automatic reminder: {from_system})"
```

Comment:
the aim of the `automatic` flag is to let the bot + user know that this is not a user generated reminder